index.html(details added)

```diff
@@ -21,22 +21,22 @@
         <tbody>
             <!--           first person -->
             <tr>
-                <td>Firstname Lastname</td>
-                <td>Artist Name</td>
-                <td>State Name</td>
-                <td>Number of Siblings</td>
+                <td>Adeshina Badmus</td>
+                <td>Femi Falana</td>
+                <td>Lagos State</td>
+                <td>3</td>
                 <td><a class="btn btn__male">Male</a></td>
             </tr>
             <!--            second person -->
             <tr>
```

Comment:
Hi @Adesanza, you are on the right track, however, you need to on fill the details for yourself once, not twice.

Secondly, because another individual's pull request as been merged while yours was waiting to be reviewed, you would have to `git fetch` to have your repository in sync with the current version. Let me know when you've done this.